deployment/v3/utils/prop_migrator/file_comparator.py

```diff
@@ -0,0 +1,135 @@
+import os
+import subprocess
+import csv
+
+# Prompt for existing Old configuration repo URL and branch
+old_repo_url = input("Enter existing old configuration repo URL: ")
+old_repo_branch = input("Enter existing old configuration repo branch: ")
+
+# Create Old dir
+old_dir = "old-config"
+os.makedirs(old_dir, exist_ok=True)
+
+# Clone the existing repo with Old-repo-branch inside the old directory
+old_repo_path = os.path.join(old_dir, old_repo_branch)
+subprocess.run(["git", "clone", "--branch", old_repo_branch, old_repo_url, old_repo_path])
+
+# Prompt for New based configurations URL and branch
```

Comment:
replace new with latest in all these places.

Reply:
done